Sprint-2/implement/contains.test.js

```diff
@@ -20,16 +20,33 @@ as the object doesn't contains a key of 'c'
 // Given an empty object
 // When passed to contains
 // Then it should return false
-test.todo("contains on empty object returns false");
+describe("contains",()=>{
```

Comment:
Looks good, for all describe , I would use what has been suggested for describing your tests: 

```
describe(" Given a contains function" , () => {
  test(" When passed an empty object to contains, Then it should return false" , () => {
     expect(...)
  })

} )

```

Reply:
Thanks for the feedback, I appreciate it. I updated the code accordingly